docs/v0.5.x/plugin-packs.md

```diff
@@ -0,0 +1,106 @@
+
+To make it easier to get up and running with this deployment strategy, you could instead just install the [ember-cli-deploy-lightning-pack](https://github.com/ember-cli-deploy/ember-cli-deploy-lightning-pack). This pack will install the above plugins automatically
+and provide you with a blueprint `config/deploy.js`.
+
+### How do I use a Plugin Pack?
+
+A plugin pack is just an ember-cli addon so you just need to install it like you would any addon:
+
+```bash
+ember install ember-cli-deploy-lightning-pack
+```
+
+From then on, just carry on deploying with [The Lightning Strategy](../the-lightning-strategy).
+
+### Why would I create a Plugin Pack?
+
+There are a couple of reason why you might want to create your own plugin pack.
+
+The first might be that you want to package up your deployment strategy so it's easy for you to implement across multiple ember-cli projects. This would save you the time of having to install all of the
+required plugins in every project.
+
+The second reason you might create a plugin pack is to contribute your deployment strategy ideas back to the ember-cli-deploy community. By publishing a plugin pack, you are telling the community that you believe you know of a
+good deployment strategy for ember-cli apps and you are showing them how they can implement that strategy with minimal effort.
+
+Both of these reasons come down to making it as easy as possible to get deploying quickly.
+
+### So how do I create a Plugin Pack?
+
+It's easy! As mentioned before, a plugin pack is simply an ember-cli addon. All you need to do is this:
+
+First create an ember-cli-addon:
+
+```bash
+ember new ember-cli-deploy-my-plugin-pack
+```
+
+Then you want to add the plugin pack keyword to `package.json` like this:
+
+```javascript
+// package.json
+
+"keywords": [
+  "ember-addon",
+  "ember-cli-deploy-plugin-pack"
+]
+
+```
+
+Finally you will need to add the plugins that you would like this plugin pack to install, to the the package.json:
+
+```javascript
+// package.json
+
+"dependencies": {
+  "ember-cli-deploy-build": "0.1.0-beta.1",
+  "ember-cli-deploy-s3": "0.1.0-beta.1",
+  "ember-cli-deploy-redis": "0.1.0-beta.1"
+}
+
+```
+
+Then if you would like to be a good citizen of the ember-cli-deploy world, you might create a blueprint to generate an example `config/deploy.js` for this particular plugin setup.
+See the [ember-cli-deploy-lightning-pack blueprint](https://github.com/ember-cli-deploy/ember-cli-deploy-lightning-pack/tree/master/blueprints/lightning-deploy-config) as an example.
+
+
+### Core Plugin Packs
+
+The following plugin packs are maintained by the ember-cli-deploy core team:
+
+- [ember-cli-deploy-lightning-pack](https://github.com/ember-cli-deploy/ember-cli-deploy-lightning-pack)
+
+
+### Community Maintained Plugin Packs
+
+The following plugin packs are deeloped by community members:
+
+- [ember-pagefront](https://github.com/pagefront/ember-pagefront)
```

Comment:
https://github.com/duizendnegen/ember-cli-deploy-azure/tree/feature/0.5.0 is, it's wrapping -tables and -blob (and others) - I'll merge into master once 0.5.0 also lands - that'd make sure that `npm install ember-cli-deploy-azure` works for 0.4 and 0.5 just the same.